Marlin/src/feature/pause.cpp

```diff
@@ -252,94 +256,143 @@ void wait_for_filament_reload(const int8_t max_beep_count/*=0*/) {
 }
 
 void resume_print(const float &load_length/*=0*/, const float &initial_extrude_length/*=0*/, const int8_t max_beep_count/*=0*/) {
-  bool nozzle_timed_out = false;
-
   if (!move_away_flag) return;
-
+  bool nozzle_timed_out = false;
+  bool swap_spool_disabled = true;
+  int16_t fansp=fanSpeeds[PAUSE_FAN];
+  
+  #if ENABLED(PAUSE_SPOOL_SWAP)
+	swap_spool_disabled= !swap_spool_enabled ; // only if next tool option enabled in lcd or gcode //steeve
+  #endif
+  
```

Comment:
i have created  swap spool disabled , because the next code , need to inhibit all menus
This boolean is installed in the general codebase , and is the switch for the menu
If swap spool compiled , this switch can be turned off and if not compiled always on
If you have a better idea may be